source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
Sorry, just recalled there’s also #121 which I’m working on in the background. I guess we can leave all the refactoring stuff to that PR because 1) I’d do them there anyway and 2) more changes will mean more merge conflicts.

I suggest to not change anything here.